src/main/webapp/templates/docs/uploading-a-build.html

```diff
@@ -71,7 +71,7 @@
             <arg value="-d" />
             <arg value="@./target/cucumber-report.json" />
             <arg
-              value="https://ohp-xbdd:8443/xbdd/rest/reports/${project.artifactId}/${version.major}.${version.minor}.${version.servicepack}/${buildNumber}" />
+              value="https://\{{YOUR-XBDD-CONTEXT-ROOT}}/rest/reports/${project.artifactId}/${version.major}.${version.minor}.${version.servicepack}/${buildNumber}" />
```

Comment:
Slash is required to escape, otherwise the content of the brackets are not displayed